mods/ra/maps/soviet-soldier-volkov-n-chitzkoi/scu35ea.lua

```diff
@@ -0,0 +1,380 @@
+		if unit.Owner == player then
+			local MineSoldierCamera1 = Actor.Create("camera", true, { Owner = player, Location = waypoint96.Location })
+			Trigger.AfterDelay(DateTime.Seconds(10), function()
+				if MineSoldierCamera1.IsInWorld then MineSoldierCamera1.Destroy() end
+			end)
+			if not MineSoldier01.IsDead then
+				MineSoldier01.Patrol({ waypoint91.Location, waypoint95.Location, waypoint76.Location, waypoint93.Location }, false, 0)
+			end
+			if not MineSoldier02.IsDead then
+				MineSoldier02.Patrol({ waypoint92.Location, waypoint91.Location, waypoint76.Location, waypoint93.Location }, false, 0)
+			end
+			if not MineSoldier03.IsDead then
+				MineSoldier03.Patrol({ waypoint91.Location, waypoint95.Location, waypoint76.Location, waypoint93.Location }, false, 0)
+			end
+			if not MineSoldier04.IsDead then
+				MineSoldier04.Patrol({ waypoint92.Location, waypoint95.Location, waypoint76.Location, waypoint93.Location }, false, 0)
+			end
+			if not MineSoldier05.IsDead then
+				MineSoldier05.Patrol({ waypoint90.Location, waypoint91.Location, waypoint95.Location, waypoint76.Location, waypoint93.Location }, false, 0)
+			end
+			if not MineSoldier06.IsDead then
+				MineSoldier06.Patrol({ waypoint92.Location, waypoint91.Location, waypoint93.Location }, false, 0)
+			end
+			Trigger.RemoveFootprintTrigger(id)
+		end
+	end)
+
+	Trigger.OnEnteredFootprint(MineRevealTrigger, function(unit, id)
+		if unit.Owner == goodguy then
+			local MineSoldierCamera2 = Actor.Create("camera", true, { Owner = player, Location = waypoint76.Location })
+			Trigger.AfterDelay(DateTime.Seconds(12), function()
+				if MineSoldierCamera2.IsInWorld then MineSoldierCamera2.Destroy() end
+			end)
+			Trigger.RemoveFootprintTrigger(id)
+		end
+	end)
+
+--Paradrop Rifle Team Setup
+	Trigger.OnEnteredFootprint(ParaTrigger, function(unit, id)
+		if unit.Owner == player then
+			local powerproxy = Actor.Create("powerproxy.pararifles", true, { Owner = greece })
+			local paratroopers = powerproxy.SendParatroopers(waypoint89.CenterPosition, false, Facing.South)
+			local prtcamera = Actor.Create("camera", true, { Owner = player, Location = waypoint89.Location })
+			Trigger.AfterDelay(DateTime.Seconds(10), function()
+				if prtcamera.IsInWorld then prtcamera.Destroy() end
+			end)
+			Utils.Do(paratroopers, function(unit)
+				if not unit.IsDead then
+					Trigger.OnIdle(unit, function(unit)
+						if unit.IsInWorld then
+							unit.Hunt()
+						end
+					end)
+				end
+			end)
+			if Map.LobbyOption("difficulty") == "hard" then
+				Trigger.ClearAll(RiflemanGuard01)
+				ProduceInfantry() --Greece will start infantry production right away if the difficulty is set to hard
+			end
+			Trigger.RemoveFootprintTrigger(id)
+		end
+	end)
+
+	Trigger.OnKilled(RiflemanGuard01, function()
+		ProduceInfantry() --Greece will start infantry production once this unit is dead just like in the original mission
+	end)
+
+--Kill the ore trucks when the two allied refinery are destroyed
```

Comment:
Why is this necessary? (It looks a bit odd to me.)

Reply:
The vanilla mission work that way.

Reply:
Can you please remove it (still)? I would consider that an improvement over the original mission. (It does not make sense to me why the ore trucks should blow up. As player, I would probably be rather confused by this.)